ui/helpers/utils/collectibles.js

```diff
@@ -0,0 +1,3 @@
+export const getCollectibleImageAlt = ({ name, tokenId, description }) => {
```

Comment:
Can you append some tests to this function?

Reply:
++, some basic tests would be good here.

Reply:
@DDDDDanica @darkwing I've added some basic tests, thanks!